u8timeseries/utils/timeseries_generation.py

```diff
@@ -0,0 +1,137 @@
+import numpy as np
+import pandas as pd
+import math
+from pandas.tseries.offsets import DateOffset
+from pandas.tseries.frequencies import to_offset
+from pandas.tseries.holiday import USFederalHolidayCalendar
+
+from u8timeseries.timeseries import TimeSeries
+
+
+
+def constant_timeseries(value: float = 0, length: int = 10, offset: str = 'D',
+                        start_date: pd.Timestamp = pd.Timestamp('2000-01-01')) -> 'TimeSeries':
+    """
+    Creates a constant TimeSeries with the given value, length, start date and offset.
+
+    :param value: The constant value that the TimeSeries object will assume at every index.
+    :param length: The length of the returned TimeSeries.
+    :param offset: The time differene between two adjacent entries in the returned TimeSeries. A DateOffset alias is expected;
+                   see: https://pandas.pydata.org/pandas-docs/stable/user_guide/TimeSeries.html#dateoffset-objects.
+    :param start_date: The time index of the first entry in the returned TimeSeries.
+    :return: A constant TimeSeries with value 'value'.
+    """
+
+    times = pd.date_range(periods=length, freq=offset, start=start_date)
+    values = np.full(length, value)
+
+    return TimeSeries.from_times_and_values(times, values)
+
+
+def linear_timeseries(start_value: float = 0, value_delta: float = 1, length: int = 10, offset: str = 'D',
```

Comment:
how about specifying `end_value` instead of `value_delta` ?
`value_delta` has the potential to be slightly misleading if one doesn't read the doc carefully, as it could be either the delta from one point to the next, or the delta between the first and last point of the series.